app/uk/gov/hmrc/gform/sharedmodel/formtemplate/BooleanExpr.scala

```diff
@@ -35,6 +35,7 @@ final case class Or(left: BooleanExpr, right: BooleanExpr) extends BooleanExpr
 final case class And(left: BooleanExpr, right: BooleanExpr) extends BooleanExpr
 final case object IsTrue extends BooleanExpr
 final case object IsFalse extends BooleanExpr
+final case class Includes(multiValueField: Expr, value: Expr) extends BooleanExpr
```

Comment:
We should make `multiValueField` of type `FormCtx` (as you say) to reject invalid expressions. See previous comment as to how.